eth_account/internal/transactions.py

```diff
@@ -66,12 +66,18 @@ def is_int_or_prefixed_hexstr(val):
 def is_empty_or_checksum_address(val):
     if val in {None, b'', ''}:
         return True
+    elif is_valid_address_bytes(val):
+        return True
     elif is_checksum_address(val):
         return True
     else:
         return False
 
 
+def is_valid_address_bytes(val):
```

Comment:
This can be replaced by `is_binary_address` from `eth_utils`: 
`from eth_utils import is_binary_address `

Reply:
Ah, good call! Will update that.

Reply:
Fixed: https://github.com/ethereum/eth-account/pull/36/commits/139b9b77a1b9327daad724f192bbd96786600b32